### ISSUE TITLE
TTF for Featherity

### ISSUE THREAD

**Issue by yanquisalexander** [lucide-icons/lucide#54]
Hi!

Is there any chance that Featherity can be compiled to TTF?

Thanks :hugs:

**Comment by mishkaio** [comment]
There may be a different way to do this but I can do this through an icon library tool I use called Nucleo. This includes eot, ttf, woff, woff2.
[Nucleo-Fonts.zip](https://github.com/featherity/featherity/files/4969349/Nucleo-Fonts.zip)

**Comment by ericfennis** [comment]
@rfbezerra created a script for generating Font files, maybe we check it out.
https://github.com/feathericons/feather/pull/608

**Comment by jletey** [comment]
@mishkaio That's really neat!

@ericfennis I don't have any of those tools installed locally, but we should be able to port it no problem 😄

**Comment by rfbezerra** [comment]
Well, today I tried to build Featherity, to tell you if it works or not.
Maybe the tools have evolved since, so I'm able to build with 3 tools (without inkscape):
- fontforge
- sfnt2woff (npm)
- fontcustom

It was build with success, with only one icon problem (icon-phone-call).
The same process worked well with Feather.

If you want to test, after install these tools, do a simple:
`fontcustom compile ./icons -h -n Featherity -o build -F`

**Comment by rfbezerra** [comment]
I made a docker image to simplicate necessary steps to build a TTF font.
If u want to check, it is this project [svg-to-ttf](https://github.com/rfbezerra/svg-to-ttf).
With that, its simple to build any ttf font from svg glyphs, e.g.:
```shell
docker run --rm -v "${PWD}":/fonts rfbezerra/svg-to-ttf -i ./icons -n Feather -u $(id -u)
```

**Comment by ericfennis** [comment]
@rfbezerra Sorry for the late reply. I was on vacation.

Thanks for your work on this! Looks awesome.
I'm currently working on the NPM package for featherity
First I want to release the package, including the subpackages for Vue and React, after that I will pick this up!

**Comment by yanquisalexander** [comment]
Just to know

Does the Workflow work correctly? 😅

**Comment by ericfennis** [comment]
I forgot to check, but it is working correctly!!
Link of the [artifact](https://github.com/featherity/featherity/suites/1202031238/artifacts/17790541).
I didn't test it yet.

But I don't know how we can publish this nicely?
My suggestion, maybe we need to add this to the new package, then we can serve it through unpkg.

**Comment by ericfennis** [comment]
Fixed by #100
Shortly I will create an solution for to download the icons
